serialization.h

```diff
@@ -35,6 +46,39 @@ struct ClientResponse
     uint32_t bufferPos;    // 4
     uint32_t fpsDrawing;   // 4
     uint32_t watts;        // 4
+} __attribute__((packed)); // Packed attribute required for network protocol compatibility
+
+struct ClientResponse
```

Comment:
Realistically once I know there are no v34 chips in the field, this can go away.  If I was really worried about backwards compat I wouldn't have stuck the field in the middle :-). But that's where it belongs.